tempesta_fw/t/functional/tests.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+__author__ = 'NatSys Lab'
+__copyright__ = 'Copyright (C) 2016 NatSys Lab. (info@natsys-lab.com).'
+__license__ = 'GPL2'
+
+
+import pkgutil
+import tests
+import subprocess
+import sys
+
+sys.path.append('/usr/src/projects/tempesta/tempesta_fw/t/functional/tests/helpers')
```

Comment:
The path is typically wrong: this is just your local path. Please use relative paths instead.